tbtc-ts/test/deposit.test.ts

```diff
@@ -27,18 +29,32 @@ describe("Deposit", () => {
     createdAt: 1640181600, // 22-12-2021 14:00:00 UTC
   }
 
-  // HEX of the expected deposit transaction made using the code from
+  // HEX of the expected P2WSH deposit transaction made using the code from
+  // deposit.ts and given depositData. It can be decoded with:
+  // https://live.blockcypher.com/btc-testnet/decodetx.
```

Comment:
Honestly, I'd rather avoid that. First, the link I mentioned is not the only decoder we can use so even if it goes down we will be still able to decode that transaction since it's a common thing in bitcoin. Secondly, the outcome of decoding often contains specific information added by the decoder itself and not implied from the transaction hex. Examples of such information are fields like: `preference`, `relayed_by`, etc. This can make a false impression that everything comes from transaction hex. Last but not least, I think it will decrease readability and will be another thing we need to change in case the expected transaction hex will change for whatever reason, for example, if we change something in the code assembling the deposit script.

Reply:
I am fine about that. Looking at the decoded transaction helped me to follow the code but it's not a must-have and I can see decoder-specific fields could be confusing. Resolving!